packages/tc-schema-sdk/data-accessors/type.js

```diff
@@ -133,25 +133,53 @@ const getType = function(
 	}
 
 	if (withRelationships) {
-		Object.entries(typeSchema.properties).forEach(([propName, def]) => {
-			if (def.relationship || def.cypher) {
-				if (def.hidden) {
-					delete typeSchema.properties[propName];
+		Object.keys(typeSchema.properties).forEach(key => {
+			let definition;
+			try {
+				definition = this.getRelationshipType(typeSchema.name, key, {
+					primitiveTypes,
+					includeMetaFields,
+				});
+			} catch (error) {
+				definition = typeSchema.properties[key];
+			}
```

Comment:
In these lines, we can accept both of rich relationship and current implementation